requirements.txt

```diff
@@ -1,5 +1,5 @@
 alembic==1.4.0
-google-auth==1.11.0
+google-auth==1.14.0
```

Comment:
This is ok, worst case we revert. Our integration test should atleast test this.

Reply:
Not sure it would catch, i don't think our tests cover any code that use this library.

Reply:
Actually test_integration_runner should cover this code (but it doesn't run in CI (yet)). Running it locally.

Reply:
Now that I think about it won't touch the auth code because even though it talks to gcs it uses gsutil. The test pass after I fixed some unrelated issues related to variants.